framework-plugins/lisk-framework-forger-plugin/test/utils/application.ts

```diff
@@ -35,10 +39,19 @@ export const createApplication = async (
 		network: {
 			maxInboundConnections: 0,
 		},
-	} as Partial<ApplicationConfig>;
+		plugins: {
+			httpApi: {
```

Comment:
http api is not needed in this plugin

Reply:
its required.. i am using it to send transactions..

Reply:
As highlighted earlier, you can use `app:postTransaction` to directly send transactions to application. This will help to use second plugin as dependency.